released/packages/coq-dijkstra/coq-dijkstra.0.1.0/opam

```diff
@@ -0,0 +1,40 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "A Verified Implementation of Dijkstra's Algorithm"
+maintainer: ["Masayuki Mizuno <mizuno@wantedly.com>"]
+authors: ["Masayuki Mizuno <mizuno@wantedly.com>"]
+license: "MIT"
+homepage: "https://github.com/fetburner/coq-dijkstra"
+bug-reports: "https://github.com/fetburner/coq-dijkstra/issues"
+depends: [
+  "dune" {>= "2.5"}
+  "coq" {>= "8.8"}
+  "coq-mathcomp-ssreflect" {>= "1.11"}
+]
+build: [
+  ["dune" "subst"] {pinned}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/fetburner/coq-dijkstra.git"
+tags: [
+  "keyword:graph theory"
+  "keyword:shortest path"
+  "keyword:Dijkstra's algorithm"
+  "category:Graph Theory"
+  "date:2021-03-02"
+  "logpath:dijkstra"
```

Comment:
Is that the actual installation path? I don't know where Dune installs the files by default, so maybe this is fine. But I am pointing it out in case it should be `"logpath:Dijkstra"` or something else.

Reply:
As you mentioned, the correct installation path was `logpath:Dijkstra`. I am sorry for taking up your time.
https://github.com/fetburner/coq-dijkstra/blob/1ca9e67958e61f7159c46206720e0013a4e5e3c3/theories/dune#L2
https://dune.readthedocs.io/en/stable/dune-files.html#coq-theory